tools/topology/topology2/cavs-nocodec.conf

```diff
@@ -165,13 +285,45 @@ Object.PCM {
 		}
 	}
 	pcm."1" {
-		name	"DMIC"
+		name	"Port1"
```

Comment:
Why is dmic being moved to PCM3 ? We should note the reason in the commit message.

Reply:
@lgirdwood - Updated the commit message